-1

**Failed Tests**:   Build 
**Summary**: https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-a953b4/39912/summary.html
**COMMIT**: bd11cb5d921dc22cea3b680344217e644d140816
**CMSSW**: CMSSW_14_0_X_2024-06-16-2300/el8_amd64_gcc12
**Additional Tests**: NANO
**User test area**: For local testing, you can use `/cvmfs/cms-ci.cern.ch/week0/cms-sw/cmssw/45176/39912/install.sh` to create a dev area with all the needed externals and cmssw changes.


## Build

I found compilation error when building: 

<pre>
gmake[1]: *** [config/SCRAM/GMake/Makefile.rules:1997: CompilePython] Error 1
>> Plugins of all types refreshed.
gmake[1]: Target 'PostBuild' not remade because of errors.
gmake[1]: Leaving directory '/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_14_0_X_2024-06-16-2300'
gmake: *** [config/SCRAM/GMake/Makefile.rules:1883: src] Error 2
gmake: Target 'all' not remade because of errors.
gmake: *** [There are compilation/build errors. Please see the detail log above.] Error 2
+ eval scram build outputlog '&&' '(python3' /data/cmsbld/jenkins/workspace/ib-run-pr-tests/cms-bot/buildLogAnalyzer.py --logDir /data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_14_0_X_2024-06-16-2300/tmp/el8_amd64_gcc12/cache/log/src '||' 'true)'
++ scram build outputlog
>> Entering Package Configuration/PyReleaseValidation
</pre>

--- a/Configuration/PyReleaseValidation/python/relval_nano.py
+++ b/Configuration/PyReleaseValidation/python/relval_nano.py
@@ -214,7 +214,7 @@ steps['lepTrackInfoNANO_mc13.2']=merge([{'-s' : 'NANO:@LepTrackInfo,DQM:@nanoAOD
                                         '-n' : '1000'},
                                        steps['NANO_mc13.2']])
 
-steps['jmeNANO_rePuppi_mc13.2']merge([{'-s' : 'NANO:@JMErePuppi ', '-n' : '1000'},
+steps['jmeNANO_rePuppi_mc13.2']=merge([{'-s' : 'NANO:@JMErePuppi ', '-n' : '1000'},
                                      steps['NANO_mc13.2']])
 
 

